staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -820,6 +820,11 @@ type KubeletConfiguration struct {
 	// If not specified, the value in containerRuntimeEndpoint is used.
 	// +optional
 	ImageServiceEndpoint string `json:"imageServiceEndpoint,omitempty"`
+
+	// PodresourcesLimits specifies the rate limits for the node-local podresources API endpoint
+	// +featureGate=KubeletPodResources
+	// +optional
+	PodresourcesLimits RateLimitConfiguration `json:"podresourcesLimits,omitempty"`
```

Comment:
this needs validating in ValidateKubeletConfiguration:
* must not be set if the podresources feature is disabled
* qps / burst may not be negative

Reply:
thanks for pointing this out. Fixed.